arch/cpu/cc26xx-cc13xx/Makefile.cc26xx-cc13xx

```diff
@@ -1,13 +1,8 @@
 CPU_ABS_PATH       = arch/cpu/cc26xx-cc13xx
 TI_XXWARE = $(CONTIKI_CPU)/$(TI_XXWARE_PATH)
 
-ifeq (,$(wildcard $(TI_XXWARE)))
-    $(warning $(TI_XXWARE) does not exist.)
-    $(warning Did you run 'git submodule update --init' ?)
-    $(error "")
-endif
```

Comment:
These lines need to stay. Presumably you deleted them by accident during conflict resolution